client/components/layout/NavBar.vue

```diff
@@ -1,51 +1,59 @@
 <template>
-  <nav>
-    <section>
-    <header><h2>
-      <router-link to="/">Health Journal</router-link>
-    </h2></header>
-    <ul>
-      <li>
-        <router-link to="/">Logs</router-link>
-      </li>
-      <li>
-        <router-link to="/">Trends</router-link>
-      </li>
-    </ul>
-    </section>
-    <hr/>
-    <section>
-    <header><h2>
-      <router-link to="/">Health Book</router-link>
-    </h2></header>
-    <ul>
-      <li>
-        <router-link to="/">Contacts</router-link>
-      </li>
-      <li>
-        <router-link to="/">Medications</router-link>
-      </li>
-      <li>
-        <router-link to="/">Insurance</router-link>
-      </li>
-    </ul>
-    </section>
-    <hr/>
-    <section>
-    <header><h2>
-      <router-link to="/account">My Account</router-link>
-    </h2></header>
-    </section>
-  </nav>
+  <article>
+    <nav>
+      <section>
+      <header><h2>
+        <router-link to="/">Health Journal</router-link>
```

Comment:
I don't believe users that are not logged in have access to any of these pages except 'My Account' – might have to add a v-if="$store.state.username"

Reply:
out of scope for this PR. We can discuss more once we have more frontend to talk about.